files/scripts/jupyter_migration.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+
+# We need to list all the existing volumes related to Jupyter, usually named pvc-"[Aa0-zZ9]?-*", and we have to rule out the one for the deployment.
+# Then we should create snapshots for those volumens 
+# kubectl get persistentvolume --kubeconfig fauziv1/kubeconfig-KUBE_AWS -o name |cut -d/ -f2 |grep -v pvc-5a172428-d643-11e8-82ba-0e3bbdfa5f60 |while read i; do aws ec2 create-snapshot --description "SS-${i}" --tag-specifications "ResourceType=snapshot,Tags=[{Key=Name,Value=${i}}]" --volume-id $(kubectl --kubeconfig fauziv1/kubeconfig-KUBE_AWS get persistentvolume -o yaml --export $i |yq -r .spec.awsElasticBlockStore.volumeID | awk -F/ '{print $NF}'); done
+
+# kubectl get persistentvolume -o name |cut -d/ -f2 |grep -v pvc-47b422cc-854a-11e8-84de-0ec14f5e5db4 |while read i; do aws ec2 create-snapshot --description "SS-${i}" --tag-specifications "ResourceType=snapshot,Tags=[{Key=Name,Value=${i}}]" --volume-id $(kubectl get persistentvolume -o yaml --export $i |yq -r .spec.awsElasticBlockStore.volumeID | awk -F/ '{print $NF}'); done
+
+AZs=("a" "c" "d")
+COUNTER=0
+
+# When we have snapshots of the active volumes, we must restore them so the hub can find them respectively.
+SNAPSHOTS=$(aws ec2 describe-snapshots --filters "Name=description,Values=SS*" --query "Snapshots[].SnapshotId" --output text)
+
+for SNAP in ${SNAPSHOTS};
```

Comment:
So we're scanning a bunch of snapshots, creating a volume from each, and a persistent-volume.yaml for each?